### DISCUSSION TITLE
How to set `n` as default value when type `lazygit` in a not git location.

### DISCUSSION THREAD

**Discussion by LintaoAmons** [jesseduffield/lazygit#1902] [Q&A] [answered]
When I type `lazygit` in a path that is not a git repo. 

The prompt `Not in a git repository. Create a new git repository? (y/n): ` I actually always answer `n`

So I was thinking is there any config I can use to set it not ask me anymore, and just open the app?

**Comment by venkateshdi** [comment, marked as answer]
you can use this config and set to 'skip' to open the app from any location.
[notARepository](https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#launching-not-in-a-repository-behaviour)
